055-ChaosStudio4AKS/README.md

```diff
@@ -0,0 +1,58 @@
+# What The Hack - ChaosStudio4AKS
+
+## Introduction 
+
+Azure Chaos Studio (Preview) is a managed service for improving resilience by injecting faults into your Azure applications. Running controlled fault
+injection
+experiments against your applications, a practice known as chaos engineering, helps you to measure, understand, and improve resilience against real-world
+incidents, such as a region outages or application failures causing high CPU utilization on a VMs, ScaleSets, and Azure Kubernetes.
+
+
+## Learning Objectives
+This “What the Hack” WTH is designed to introduce you to Azure Chaos Studios (Preview) and guide you through a series of hands-on challenges to accomplish
+the following:
+  
+* Leverage the Azure Chaos Studio to inject failure into an application/workload
+* Provide hands-on understanding of Chaos Engineering 
+* Understand how resilientcy can be achieved with Azure 
+
+In this WTH, you are the system owner of the Contoso Pizzeria Application (or you may bring your own application). Super Bowl Sunday is Contoso Pizza's busiest time of the year, the pizzeria
```

Comment:
Do we want partners/customers to bring their own app?  Based on the challenges I wonder if you will encounter the same experience of the hack if their app has some resiliency built in already.

Reply:
I think we had discussed this with Peter last year and he recommended we have an option to bring their own app

Reply:
@jrzyshr - Tagging to comment.